bin/migrate-oats-data/sql/insert-batch-application.sql

```diff
@@ -38,44 +62,58 @@ panel_lookup AS (
     WHERE
         oo2.organization_type_cd = 'PANEL'
 ),
--- Step X: Perform lookup to retrieve type code
+
+-- Step 4: Perform lookup to retrieve type code
+
 application_type_lookup AS (
     SELECT
         oaac.alr_application_id AS application_id,
         oacc."description" AS "description",
-        oaac.alr_change_code AS code,
-        acl.alcs_code AS alcs_code
-
+        oaac.alr_change_code AS code
     FROM
         oats.oats_alr_appl_components AS oaac
-        JOIN oats.oats_alr_change_codes oacc ON oaac.alr_change_code = oacc.alr_change_code
-        JOIN oats.appl_code_lut AS acl ON oaac.alr_change_code = acl.oats_code
-
-        
+        JOIN oats.oats_alr_change_codes oacc ON oaac.alr_change_code = oacc.alr_change_code   
 )
 
--- Step 3: Insert new records into the alcs_applications table
+-- Step 5: Insert new records into the alcs_applications table
 SELECT
     oa.alr_application_id :: text AS file_number,
-    -- TODO: type code lookup
-    'NARU' as type_code,
-    -- COALESCE(atl.alcs_code, 'NARU') AS type_code,
+
+    CASE
+        WHEN atl.code = 'TUR' THEN 'TURP'
+        WHEN atl.code = 'INC' THEN 'INCL'
+        WHEN atl.code = 'EXC' THEN 'EXCL'
+        WHEN atl.code = 'SDV' THEN 'SUBD'
+        WHEN atl.code = 'NFU' THEN 'NFUP'
+        WHEN atl.code = 'SCH' THEN 'PFRS'
+        WHEN atl.code = 'EXT' THEN 'ROSO'
+        WHEN atl.code = 'FILL' THEN 'POFO'
+        -- WHEN atl.code = 'SRW' THEN 'NARU'
+        -- WHEN atl.code = 'CSC' THEN 'NARU'
+        WHEN atl.code = 'NAR' THEN 'NARU'
+        ELSE 'NARU'
+    END AS type_code,
     CASE
         WHEN applicant_lookup.orgs IS NOT NULL THEN applicant_lookup.orgs
         WHEN applicant_lookup.persons IS NOT NULL THEN applicant_lookup.persons
         ELSE 'Unknown'
     END AS applicant,
-    -- TODO: panel region lookup
-    -- 'INTR',
 	ar.code as region_code,
-    --Peace river TODO: local government lookup
-    '001cfdad-bc6e-4d25-9294-1550603da980',
+    --IN PROGRESS: local government lookup name mismatches
+    --Currently using peace river as default uuid
+    CASE
+        WHEN alcs_gov.gov_uuid IS NOT NULL THEN alcs_gov.gov_uuid
+        ELSE '001cfdad-bc6e-4d25-9294-1550603da980'
```

Comment:
Is it because LG is a required field on application?

Reply:
It is required to be a uuid, you had requested that it was not left to be NULL

Reply:
I meant is it a nullable field in DB? If there is no way to determine what LG it belongs to and the field is not nullable then you need to set that value. Otherwise, it should be null.